src/backend/templates/mirrors.html

```diff
@@ -94,6 +94,11 @@
                             {% endif %}
                         </td>
                     {% endfor %}
+                    <td>
+                        {% if mirror.ipv6 %}
```

Comment:
Did you add the column to header?

Reply:
Yep https://github.com/AlmaLinux/mirrors/pull/291/commits/983abaf6198191a924e5546e6e6b29393c3df0b8#diff-7dc0ea0263421b070b14c289b52f9467996fb61796dbd1277b1c99ea1da14e4fR143